test/if.sh

```diff
@@ -33,8 +33,17 @@
 
```

Comment:
small note after rename: https://github.com/benjaoming/kolibri/blob/sdist-with-static-deps/test/if.sh#L8 still refers to `testif.sh`